pkg/virtctl/imageupload/imageupload.go

```diff
@@ -374,6 +376,11 @@ func createUploadPVC(client kubernetes.Interface, namespace, name, size, storage
 		pvc.Spec.AccessModes = []v1.PersistentVolumeAccessMode{v1.PersistentVolumeAccessMode(accessMode)}
 	}
 
+	if blockVolume {
+		volMode := v1.PersistentVolumeBlock
+		pvc.Spec.VolumeMode = &volMode
```

Comment:
Can you add a simple test in `tests/imageupload_test.go` to verify that the PVC is created the  way we want it?

Reply:
Of course! That is done now too.